packages/react-form/src/hooks/list/dynamiclistreset.ts

```diff
@@ -0,0 +1,18 @@
+import {useCallback, useRef} from 'react';
+
+import {DynamicListBag} from '../../types';
+
+export function useDynamicListReset(lists?: DynamicListBag) {
```

Comment:
Happy to see this stuff broken up into composable hooks, very in-keeping with the spirit of the existing APIs :)